packages/binding-builder-cli/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "@rspack/binding-builder-cli",
+  "version": "1.4.2",
+  "description": "CLI for rspack binding builder",
+  "homepage": "https://rspack.rs",
+  "bugs": "https://github.com/web-infra-dev/rspack/issues",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/web-infra-dev/rspack",
+    "directory": "packages/binding-builder-cli"
+  },
+  "license": "MIT",
```

Comment:
The new package should contain a license file: https://github.com/web-infra-dev/rspack/blob/main/packages/rspack/LICENSE

Reply:
Done